.docker/Dockerfile

```diff
@@ -7,10 +7,12 @@ RUN groupmod -g 1000 www-data \
 COPY /wait-for-it.sh /tmp/
 COPY /docker_run_git.sh /tmp/
 
+RUN mkdir -p /var/www/.composer
 RUN mkdir -p /var/www/.npm
-RUN chown -R www-data:www-data /var/www/.npm
 
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN chown -R www-data:www-data /var/www/
```

Comment:
Not sure we want to change the parent directory as we only want to change .composer and .npm paths, Seems I made the same PR as you #25940 :sweat_smile:

Reply:
Also because the parent directory can be mounted and the whole content is independent from the Docker fs

Reply:
I think there was a problem with building assets, couldn't write files in some directories. That's why.